packages/demos/src/node-integrator.js

```diff
@@ -0,0 +1,43 @@
+// @ts-check
+
+/**
+ * @param {string} source
+ * @return {any}
```

Comment:
Use `{{readonly default: import('./app-renderer').AppRenderer}}` and assert the type internally with e.g. `typeof mod.exports.default === 'function'`